src/main/scala/nl/knaw/dans/easy/multideposit/DDM.scala

```diff
@@ -21,24 +21,22 @@ object DDM {
   val profileFields: Dictionary =
     Map("DC_TITLE" -> "dc:title",
       "DC_DESCRIPTION" -> "dcterms:description",
-      "DC_CREATOR" -> "dc:creator",
+      "DC_CREATOR" -> "dc:creator", // TODO verwijder
```

Comment:
Removed them for now. Notification has been sent to @lindareijnhoudt. Maybe we can create a *deprecated* warning out of them when we implement the CSV validation (EASY-1015)?